conn_profile_events.go

```diff
@@ -18,12 +17,16 @@ type ProfileEvent struct {
 	Value       int64
 }
 
-func (c *connect) profileEvents(ctx context.Context) ([]ProfileEvent, error) {
+func (c *connect) profileEvents(ctx context.Context, scanEvents bool) ([]ProfileEvent, error) {
 	block, err := c.readData(ctx, proto.ServerProfileEvents, false)
 	if err != nil {
 		return nil, err
 	}
 	c.debugf("[profile events] rows=%d", block.Rows())
+	if !scanEvents {
+		c.debugf("[profile events] skipping scan")
```

Comment:
can we add also why we skip? may be something like
`[profile events] skipping scan. No profile-events handler registered`